src/types/ColorFix.cpp

```diff
@@ -1,231 +1,195 @@
+    0x1.df1ca4p-4f,  0x1.e9a122p-4f,   0x1.f447dp-4f,  0x1.ff10c2p-4f,  0x1.04fe0ep-3f,    0x1.0a85p-3f,  0x1.101d46p-3f,   0x1.15c6fp-3f,  0x1.1b820ap-3f,  0x1.214ea8p-3f,  0x1.272cd6p-3f,  0x1.2d1ca4p-3f,   0x1.331e2p-3f,  0x1.39315ap-3f,   0x1.3f566p-3f,  0x1.458d46p-3f,
+    0x1.4bd612p-3f,  0x1.5230d8p-3f,   0x1.589dap-3f,  0x1.5f1c82p-3f,  0x1.65ad88p-3f,  0x1.6c50c2p-3f,  0x1.73063cp-3f,  0x1.79ce06p-3f,  0x1.80a82cp-3f,  0x1.8794bep-3f,  0x1.8e93c8p-3f,  0x1.95a55cp-3f,  0x1.9cc984p-3f,  0x1.a4004ep-3f,  0x1.ab49ccp-3f,  0x1.b2a606p-3f,
+    0x1.ba1516p-3f,  0x1.c196f8p-3f,   0x1.c92bcp-3f,  0x1.d0d37ep-3f,   0x1.d88e4p-3f,  0x1.e05c18p-3f,  0x1.e83d08p-3f,   0x1.f0312p-3f,   0x1.f8387p-3f,  0x1.002984p-2f,  0x1.044078p-2f,  0x1.08611ap-2f,  0x1.0c8b74p-2f,  0x1.10bf8ap-2f,  0x1.14fd64p-2f,  0x1.194506p-2f,
+    0x1.1d967ap-2f,  0x1.21f1c2p-2f,  0x1.2656e8p-2f,   0x1.2ac5fp-2f,  0x1.2f3ee2p-2f,  0x1.33c1c4p-2f,  0x1.384e9cp-2f,  0x1.3ce56ep-2f,  0x1.418644p-2f,  0x1.463124p-2f,   0x1.4ae61p-2f,  0x1.4fa512p-2f,   0x1.546e3p-2f,  0x1.59416ep-2f,  0x1.5e1ed2p-2f,  0x1.630666p-2f,
+     0x1.67f83p-2f,   0x1.6cf43p-2f,  0x1.71fa6ep-2f,  0x1.770af2p-2f,  0x1.7c25c2p-2f,  0x1.814ae2p-2f,  0x1.867a5ap-2f,  0x1.8bb42ep-2f,  0x1.90f866p-2f,  0x1.964706p-2f,  0x1.9ba016p-2f,  0x1.a1039ap-2f,  0x1.a67198p-2f,  0x1.abea16p-2f,  0x1.b16d1ap-2f,  0x1.b6faa8p-2f,
+    0x1.bc92cap-2f,  0x1.c23582p-2f,  0x1.c7e2d6p-2f,  0x1.cd9acep-2f,  0x1.d35d6cp-2f,  0x1.d92abap-2f,  0x1.df02bap-2f,  0x1.e4e574p-2f,  0x1.ead2ecp-2f,  0x1.f0cb28p-2f,  0x1.f6ce2ep-2f,  0x1.fcdc04p-2f,  0x1.017a5ap-1f,  0x1.048c1cp-1f,  0x1.07a34ep-1f,  0x1.0abff2p-1f,
+    0x1.0de20cp-1f,  0x1.11099ep-1f,  0x1.1436acp-1f,  0x1.176936p-1f,   0x1.1aa14p-1f,  0x1.1ddecep-1f,  0x1.2121e2p-1f,  0x1.246a7cp-1f,  0x1.27b8a2p-1f,  0x1.2b0c56p-1f,  0x1.2e659ap-1f,  0x1.31c472p-1f,  0x1.3528dep-1f,  0x1.3892e2p-1f,  0x1.3c0282p-1f,  0x1.3f77bep-1f,
+    0x1.42f29cp-1f,  0x1.46731cp-1f,   0x1.49f94p-1f,  0x1.4d850cp-1f,  0x1.511682p-1f,  0x1.54ada6p-1f,  0x1.584a7ap-1f,    0x1.5bedp-1f,  0x1.5f953cp-1f,  0x1.63432ep-1f,  0x1.66f6d8p-1f,   0x1.6ab04p-1f,  0x1.6e6f66p-1f,  0x1.72344cp-1f,  0x1.75fef8p-1f,  0x1.79cf6ap-1f,
+    0x1.7da5a4p-1f,  0x1.8181a8p-1f,  0x1.85637cp-1f,   0x1.894b2p-1f,  0x1.8d3896p-1f,  0x1.912be2p-1f,  0x1.952504p-1f,  0x1.992402p-1f,  0x1.9d28dcp-1f,  0x1.a13396p-1f,   0x1.a5443p-1f,  0x1.a95aaep-1f,  0x1.ad7714p-1f,   0x1.b1996p-1f,  0x1.b5c19ap-1f,   0x1.b9efcp-1f,
+    0x1.be23d8p-1f,  0x1.c25ddcp-1f,  0x1.c69ddep-1f,  0x1.cae3d2p-1f,  0x1.cf2fc6p-1f,  0x1.d381acp-1f,  0x1.d7d99ap-1f,  0x1.dc377ep-1f,   0x1.e09b7p-1f,  0x1.e5055ep-1f,  0x1.e9755cp-1f,  0x1.edeb5cp-1f,  0x1.f26772p-1f,  0x1.f6e98cp-1f,   0x1.fb71cp-1f,         0x1p+0f,
+};
+// clang-format on

```

Comment:
for the record, this whole table looks insane and no human can actually read that. 

_but it works_